app/src/main/java/ch/epfl/sweng/favors/favors/FavorDetailView.java

```diff
@@ -98,6 +103,8 @@ public void onCreate(Bundle savedInstanceState) {
                 //TODO add token cost binding with new database implementation
             });
         }
+
+        mFirestore = FirebaseFirestore.getInstance();
```

Comment:
I think that would break the tests. To get an instance you should use Database.getInstance()

Reply:
Because FirebaseFirestor.getInstance() return null when we are testing offline